eng/targets/TargetFrameworks.props

```diff
@@ -15,7 +15,7 @@
     <NetRoslyn>net9.0</NetRoslyn>
     <NetRoslynAll>net8.0;net9.0</NetRoslynAll>
```

Comment:
Did this need to also be updated to contain net10.0?

Reply:
Good catch. Thanks Jason! https://github.com/dotnet/roslyn/pull/81545